### PR TITLE
Implemented auth via angular

### PR DESCRIPTION
**Pull request by ostcar**
Also added the derective osPerms to check if the current user has permissions

fixes: #1454

### CODE REVIEW

File: openslides/urls.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 urlpatterns = patterns(
     '',
     url(r'^rest/', include(router.urls)),
-    # TODO: add "special" urls, for example pdf views etc.
+    (r'^api/users/', include('openslides.users.urls')),
```
Comment (normanjaeckel) [inline openslides/urls.py]: Use another prefix here or find a solution without any prefix as discussed.

File: openslides/utils/views.py [hunk 2]
```diff
@@ -76,14 +76,20 @@ class AjaxMixin(object):
...
+    def get_ajax_context(self, **context):
         """
         Returns a dictonary with the context for the ajax response.
         """
-        return kwargs
+        return context
 
     def ajax_get(self, request, *args, **kwargs):
         """
+        Depricated. Use ajax_response instead.
```
Comment (normanjaeckel) [inline openslides/utils/views.py]: Typo: Depr~~i~~ecated

File: openslides/users/views.py [hunk 3]
```diff
@@ -499,3 +503,54 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['user'] = self.request.user
         return kwargs
+
```
Comment (normanjaeckel) [inline openslides/users/views.py]: Please remove old login and logout views from this file and from urls.py.

File: openslides/urls.py (hunk 1 above)
Comment (ostcar) [inline openslides/urls.py]: done

File: openslides/utils/views.py (hunk 2 above)
Comment (ostcar) [inline openslides/utils/views.py]: done

File: openslides/users/views.py (hunk 3 above)
Comment (ostcar) [inline openslides/users/views.py]: done

**Comment by ostcar** [comment]
all done. I had to remove the code to get rid of the old urls.

**Pull request closed without merging**